src/main/scala/io/iohk/ethereum/merklepatriciatrie/package.scala

```diff
@@ -0,0 +1,34 @@
+package io.iohk.ethereum
+
+package object merklepatriciatrie {
+  trait ByteArraySerializable[T] {
+    def toBytes(input: T): Array[Byte]
+
+    def fromBytes(bytes: Array[Byte]): T
+  }
+
+  trait DataSource {
```

Comment:
Sure, but why `package object`? Package objects are usually used for things that cannot be placed at a top-level of a package, like vals and defs. But traits can :)